src/OnboardingSPA/components/App/index.js

```diff
@@ -165,49 +180,65 @@ const App = () => {
 		const previousStep = window.nfdOnboarding?.previousStepID;
 		if ( typeof previousStep !== 'string' ) {
 			window.nfdOnboarding.previousStepID = location.pathname;
-			HiiveAnalytics.dispatchEvents( HIIVE_ANALYTICS_CATEGORY );
+			HiiveAnalytics.dispatchEvents( CATEGORY );
```

Comment:
No, it will not, and this is **not** a refresh condition. If you remember, as per requirements in v1 of analytics - we need to dispatch pending events when a step changes, `handlePreviousStepTracking` is the function that meets that.